azure-pipelines.yml

```diff
@@ -476,6 +476,49 @@ jobs:
           trigger_sha: '$(trigger_sha)'
       - template: ci/report-end.yml
 
+  - job: compat_versions_pr
+    dependsOn:
+    - release
```

Comment:
Note: no need for extra condition checks here, this will only run if `release` has run.

Reply:
Since I wondered that myself: If `release` has finished we’ve uploaded to maven, github releases and the gcp bucket.